crates/bevy_pbr/src/prepass/mod.rs

```diff
@@ -313,6 +314,7 @@ impl<M: Material> FromWorld for PrepassPipeline<M> {
             view_layout_motion_vectors,
             view_layout_no_motion_vectors,
             mesh_layouts: mesh_pipeline.mesh_layouts.clone(),
+            mesh_buffer_batch_size: mesh_pipeline.per_object_buffer_batch_size,
```

Comment:
i think this would make more sense as a shader def associated with the `bevy_pbr::mesh_bindings` module directly (like `MAX_DIRECTIONAL_LIGHTS` in mesh.rs), then it could be removed from the mesh pipeline and from here, and from the specialize functions

Reply:
Cool feature! Done.